playwright/test/view-items.test.ts

```diff
@@ -103,15 +103,16 @@ describe('Scenario 3: A user wants information about the item they are viewing',
 
   test('the item has date information', async () => {
     const dates = await page.textContent(workDates);
-    expect(dates).toBe('Date1497');
+    // TODO: this text isn't very explanitory and should probably be updated in the DOM
+    expect(dates).toBe('Date1496[7]');
```

Comment:
Why/how has this changed since the test was set up? Has the underlying data changed for this item?

Reply:
Nope, I think we are rendering the page number in the same span, hence the TODO. 

Sorry to have been so brash with the merge, hence asking if we could follow up though.

Reply:
That should just be the date info that shows below the title of an item (if it's available). I don't _think_ the page numbers should be anywhere near it.

Reply:
Oh weird – looks like the field has changed. In that case it's perfectly fine that the test would fail 
![image](https://user-images.githubusercontent.com/1394592/115366201-046bda00-a1bd-11eb-8eb4-42d9dfd5c5f1.png)

Reply:
Completely agree. It was also a little hard to understand why we weren't checking the active index, as that felt important - let's look at this and fix it up properly, this was a plaster to what I was expecting would be a better fix.